src/main/groovy/de/dkfz/eilslabs/batcheuphoria/TestExecutionService.groovy

```diff
@@ -55,6 +57,11 @@ class TestExecutionService implements ExecutionService {
     }
 
     @Override
+    RestResult execute(RestCommand restCommand) {
```

Comment:
The normal ExecutionService class should NOT handle RestCommand . This is specific to RestServices. Please override execute(Command) and don't create it additionally.